mattdrayer/eliminate-base-distinguished: Switch back to honor/verified

mattdrayer opened the pull request:
Per @pbaruah and @frrrances, we're removing the last bits of the base/distinguished course mode concepts and instead sticking with honor/verified, at least for now.

frrrances commented:
I think this is ready to go - no sass changes were needed as @talbs accounted for using base or honor, and verified or distinguished. :+1:

talbs commented:
@mattdrayer, there are some assets made for the base/distinguished cases (suffixed with *-base, *-distingushed) that can be removed now (since we're reverting to edX-centric terms for certs). You can find them in https://github.com/edx/edx-platform/tree/mattdrayer/eliminate-base-distinguished/lms/static/certificates/images.

Mind deleting those bits?

talbs commented:
Also, @frrrances and @mattdrayer, if we're not supporting base and distinguished terms, we can remove the following Sass selectors:

* https://github.com/edx/edx-platform/blob/mattdrayer/eliminate-base-distinguished/lms/static/certificates/sass/_components.scss#L460
* https://github.com/edx/edx-platform/blob/mattdrayer/eliminate-base-distinguished/lms/static/certificates/sass/_components.scss#L486

mattdrayer commented:
@talbs @frrrances -- it's been a while and I'd like to revisit this PR -- I've rebased the branch onto the tip of master and squashed the two commits into one.  Can you both take a quick look now and let me know what else needs to be done?  In particular I'm wondering about Brian's comments regarding removal of certain Sass items -- Frances, is that something you can help with?  Maybe add a commit to this PR?

mattdrayer commented:
jenkins run python

frrrances commented:
@mattdrayer I made the changes @talbs suggested. Let me know if you have any questions. Otherwise, I think this is ready to go.

mattdrayer commented:
Looks like the branch needs to be rebased -- will take care of that now

mattdrayer commented:
jenkins run python

mattdrayer commented:
@frrrances @talbs -- Jenkins is finally nodding his head in approval, so I'm going to merge before he changes his mind :)
